### ISSUE TITLE
c-form-control -> c-select `id` ends up on select and wrapper-div (breaks label functionality / creates invalid HTML)

### ISSUE THREAD

**Issue by escapedcat** [chakra-ui/chakra-ui-vue#484]
**Describe the bug**
https://vue.chakra-ui.com/formcontrol#select-example  
Example in docs shows that `id` is being added twice to generated HTML.

**Expected behavior**
`CSelect` warpper div should not have `id="country"` only the `select` should have it.

**Screenshots**
![image](https://user-images.githubusercontent.com/1016218/132292260-eb077994-fdd3-4877-8fb0-b3e66eaa910f.png)

**Desktop (please complete the following information):**
 - OS: OSX
 - Browser: Chrome
 - Version: 92.0.4515.159 (Official Build) (x86_64)

**Comment by peoray** [comment]
Thanks for spotting that out @escapedcat. We'll look into it :)

**Comment by HerrBertling** [comment]
I think the fix would be rather easy by filtering the `data.attrs` before spreading them into the `CBox` here: https://github.com/chakra-ui/chakra-ui-vue/blob/develop/packages/chakra-ui-core/src/CSelect/CSelect.js#L34

I'd open a PR for this if that's okay @peoray?

**Comment by peoray** [comment]
Please go ahead @HerrBertling . :)

**Comment by HerrBertling** [comment]
Done :)
